UITests/ViewMemoryLeakTests.swift

```diff
@@ -99,6 +102,9 @@ class ViewMemoryLeakTests: KIFTestCase, UITextFieldDelegate {
         XCTAssertNil(webView, "webView disposed")
     }
 
+    private func getTopViewController() -> UIViewController {
+        return (UIApplication.sharedApplication().keyWindow!.rootViewController as! UINavigationController).topViewController!
```

Comment:
The window's root view controller was changed to a `UINavigationController` which was not reflected in the tests up to now.